src/cart/crt_hg_proc.c

```diff
@@ -583,7 +583,7 @@ crt_proc_in_common(crt_proc_t proc, crt_rpc_input_t *data)
 						);
 			hdr->cch_dst_tag = rpc_priv->crp_pub.cr_ep.ep_tag;
 
-			hdr->cch_src_timeout = rpc_priv->crp_timeout_sec;
+			hdr->cch_src_deadline_sec = timeout_to_deadline(rpc_priv->crp_timeout_sec);
```

Comment:
I think in general we would want to use half of the timeout value, based on what Sean had mentioned in the past, ie. client timeout value is double the server timeout.

Reply:
i am not sure using half is the right approach. i could see using "deadline - [small amount]" to account for the response delay, but for example, expiring a "60second timeout" rpc in 30 seconds (vs say 59 seconds on the receiver) would cause a lot of test issues and weird behaviors.